### ISSUE TITLE
第二版第五章mnist最佳实践样例程序代码问题

### ISSUE THREAD

**Issue by x7night** [caicloud/tensorflow-tutorial#134]
代码运行后
loss = cross_entropy_mean + tf.add_n(tf.get_collection('losses'))
显示如下异常
发生异常: ValueError
inputs must be a list of at least oneTensor/IndexedSlices with the same dtype and shape
